pkg/agent/controller/egress/egress_controller.go

```diff
@@ -467,8 +476,18 @@ func (c *EgressController) realizeEgressIP(egressName, egressIP string) (uint32,
 				return 0, fmt.Errorf("error allocating mark for IP %s: %v", egressIP, err)
 			}
 		}
+		if rateLimit != nil {
+			if c.ovsMetersSupported {
+				if err := c.installEgressQoSMeter(ipState.mark, rateLimit); err != nil {
+					return 0, err
+				}
+			} else {
+				klog.V(2).InfoS("Rate-limit in the Egress is ignored because OVS meters are not supported", "EgressName", egressName)
```

Comment:
We should perhaps make it `V(0)`, otherwise the setting won't take effect but users can know it from nowhere.

Reply:
Changed.